install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -44,38 +29,11 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 				ctx.RepoName(ctx.Config.Repository, ide.CodeIDEImage): {
 					PrePull: []string{},
 					Workdir: "/ide",
```

Comment:
it is the only reason why we keep it?

Reply:
Not sure what you mean? we still need the `InlineStatic` replacements

Reply:
ah, i see also for supervisor